### ISSUE TITLE
cw and dw revert to default vim behavior

### ISSUE THREAD

**Issue by f1sherman** [chaoren/vim-wordmotion#53]
Hi there and thanks for this super useful plugin! I'm fairly certain that the issue I'm having is specific to something in my setup, but I'm not sure what exactly is causing it. The issue is that `w` works as expected (as described in the README of this repo), but `cw`, `ce`, `dw` and `de` all revert to the default vim behavior. You can see my vimrc [here](https://github.com/f1sherman/dotvim). I'd love to hear confirmation that this is in fact not expected and any tips on how I could troubleshoot. Thank you!

**Comment by chaoren** [comment]
What's the output of `:omap w`?

It should be:
```
o  w             <Plug>WordMotion_w
```

Could you try disabling your other plugins? One of them may be interfering with the mapping.

**Comment by chaoren** [comment]
> but `cw`, `ce`, `dw` and `de` all revert to the default vim behavior.

Can you give a specific example of this? With before and after text, and the location of your cursor before entering the command.

**Comment by f1sherman** [comment]
The output of `:omap w` looks pretty suspicious to me:

```
o  w             YRMapsExpression("<SNR>45_", "w")
```

Looks like that was mapped due to [yankring.vim](https://github.com/vim-scripts/YankRing.vim), which I've apparently had installed since 2013. I removed that plugin and now it's working great, thanks!

For posterity, here is an example of what wasn't working right:

```
MyCamelCaseWord
^
```

If I typed `cw` or `dw`, the whole word would be deleted. Now it's working great, thanks!
